### ISSUE TITLE
curator profile not working

### ISSUE THREAD

**Issue by mtholder** [OpenTreeOfLife/opentree#1160]
see https://tree.opentreeoflife.org/curator/profile/sbyrum

**Comment by jimallman** [comment]
Indeed. I thought this might be due to some contributors not being indexed by **oti**, but @sbyrum clearly has lots of curation activity: https://tree.opentreeoflife.org/curator/?match=sbyrum

**Comment by jimallman** [comment]
This error occurs because @sbyrum has no assigned `username` (display name) on Github. In such a case, the curation app will add their `userid` to the curators list in Nexson, so now we follow suit. This is tested and will appear on **devtree** shortly.

NOTE that this will still fail to credit work if a curator has **changed** their username. In this case, if @sbyrum chooses a username on Github, we'll start checking for it instead, and their prior work will not be counted. At the risk of finding false positives, should we check for both username and userid?

**Comment by jimallman** [comment]
This fix is live and available on **devtree**, but it won't give a proper profile for @sbyrum there, since they have no curation activity on the dev phylesystem-0. However, I have tested this on **production** (using a temporary hot fix, now removed) and it works. I'm confident it will resolve this problem for any curator that doesn't have a display name on Github.
